packages/core-types/addon/components/field-editors/integer-editor.js

```diff
@@ -3,4 +3,16 @@ import layout from '../../templates/components/field-editors/integer-editor';
 
 export default Component.extend({
   layout,
+
+  actions: {
+    update(value) {
+      let content = this.get('content');
+      let field = this.get('field');
+      if (value == null || value === '') {
+        content.set(field, null);
+      } else {
+        content.set(field, +value);
```

Comment:
is there a reason for using `+value` over something more readable like `Number(value)`?

Reply:
I could use `Number(...)`, it's just that I saw this casting "trick" recently and really liked it. But I think you're right, as it's not as readable and widely used, it's probably better to use `Number`.